tools/format.py

```diff
@@ -48,14 +48,16 @@ def main():
         prettier()
 
 
-def qrun(cmd, env=None):
-    run(cmd, quiet=True, env=env)
+def qrun(cmd, cwd=None, env=None):
+    run(cmd, quiet=True, cwd=cwd, env=env)
 
 
 def clang_format():
     print "clang_format"
-    qrun([clang_format_path, "-i", "-style", "Google"] +
-         find_exts(["core"], [".cc", ".h"]))
+    qrun(
+        [sys.executable, clang_format_path, "-i", "-style", "Google"] +
+        find_exts(["core"], [".cc", ".h"], skip=core_symlinks),
+        env={"CHROMIUM_BUILDTOOLS_PATH": "third_party/v8/buildtools"})
```

Comment:
Digging through the `depot_tools` sources I've found out why this is necessary here. The wrapper script [expects `buildtools` to be at the top level](https://github.com/denoland/deno_third_party/blob/4b36ce22a2ef7310f815acb6a2de0d26766a24b6/depot_tools/gclient_paths.py#L101), but since we moved the symlink this is no longer the case.

So we have to either set `CHROMIUM_BUILDTOOLS_PATH` explicitly or maintain 2 `buildtools` symlinks (one at the old top level position and one inside `core/libdeno`.